test/dataframe.py

```diff
@@ -179,3 +189,1479 @@ def test_float_dataframe():
     test_abs(ray_df, pandas_df)
     test_keys(ray_df, pandas_df)
     test_transpose(ray_df, pandas_df)
+
+
+def test_add():
+    ray_df = create_test_dataframe()
```

Comment:
Wouldn't this actually raise an exception saying that `ray.init()` hasn't been called? Or is `ray.init()` being called somewhere?

Note that `ray.init()` is slow enough, and there are enough dataframe methods, that it might be better to avoid calling `ray.init()` separately for each method test.

Reply:
I think `ray.init()` is only called once for the whole script. As far as I can tell, it is fine to call `ray.init()` once in the first test and it will work for the remaining tests. Alternatively, I could pull the ray.init() out of the first test and put it at the top.